testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/tracing/TracingFeatureTestCase.java

```diff
@@ -0,0 +1,56 @@
+package org.jboss.as.test.integration.jaxrs.tracing;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.test.integration.common.HttpRequest;
+import org.jboss.as.test.integration.jaxrs.packaging.war.WebXml;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+//import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+//import java.io.File;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class TracingFeatureTestCase {
+    @Deployment
+    public static Archive<?> deploy() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "tracing.war");
+        war.addPackage(HttpRequest.class.getPackage());
+        war.addClasses(FooResource.class);
+        war.addAsWebInfResource(WebXml.get("<servlet-mapping>\n" +
+                "        <servlet-name>javax.ws.rs.core.Application</servlet-name>\n" +
+                "        <url-pattern>/tracing/*</url-pattern>\n" +
+                "    </servlet-mapping>\n" +
+                "<context-param> <param-name>resteasy.scan</param-name> <param-value>true</param-value> </context-param>" +
+                "\n"), "web.xml");
+
+
+//        war.as(ZipExporter.class).exportTo(
+//                new File("/tmp/my.war"), true);
+
+        return war;
+    }
+
+    @ArquillianResource
+    private URL url;
+
+    private String performCall(String urlPattern) throws Exception {
+        return HttpRequest.get(url + urlPattern, 10, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void testFoo() throws Exception {
+        String result = performCall("tracing/foo");
+        assertEquals("Hello World!", result);
```

Comment:
Should we not just use the REST Client here instead?

Reply:
It's just some placeholder code here and I'll do cleanup after the real test code is implemented. Sorry for the confusion :D